tests/lib/rules/comma-style.js

```diff
@@ -585,19 +585,6 @@ ruleTester.run("comma-style", rule, {
             }],
             errors: [{ messageId: "expectedCommaFirst" }]
         },
-        {
-            code: "var foo = [\n(bar\n)\n,\nbaz\n];",
-            output: "var foo = [\n(bar\n),\nbaz\n];",
-            errors: [{
-                messageId: "unexpectedLineBeforeAndAfterComma",
-                type: "Identifier"
-            }]
-        },
```

Comment:
This particular test doesn't have empty elements. If it's failing, then the change might be incorrect since it modifies behavior for arrays/patterns that don't have empty elements.

A simplified example, this time with an array pattern that doesn't have empty elements:

```js
/*eslint comma-style: ["error", "last", { 
    "exceptions": { "ArrayPattern": false }
}]*/

var [
  foo
  ,
  bar] = [];
```

This is invalid code by this rule in the actual version - [Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgY29tbWEtc3R5bGU6IFtcImVycm9yXCIsIFwibGFzdFwiLCB7IFxuICAgIFwiZXhjZXB0aW9uc1wiOiB7IFwiQXJyYXlQYXR0ZXJuXCI6IGZhbHNlIH1cbn1dKi9cblxudmFyIFtcbiAgZm9vXG4gICxcbiAgYmFyXSA9IFtdOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6Nywic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6eyJjb21tYS1zdHlsZSI6Mn0sImVudiI6e319fQ==)

After this change as it's implemented at the moment, this would be a valid code (I think, it would be nice to add a test case). Is this the desired new behavior, i.e., should we ignore any commas on separate lines?

Reply:
> Is this the desired new behavior, i.e., should we ignore any commas on separate lines?

If "comma-style" is enabled, that comma in a new line belongs either with `foo` (as default 'last'), or `bar` (if set to 'first'). Therefor your example should fail.
My attempted fix is therefor incomplete. Will revise that particular example, thanks for pointing it out.